pkg/config/config.go

```diff
@@ -713,3 +713,7 @@ func createSignedEnvelopeWithTLSBinding(
 
 	return env, nil
 }
+
+func getOrgsFromApplication(config *cb.Config) map[string]*cb.ConfigGroup {
```

Comment:
nit: Given this helper is specific to application orgs we could move this application.go.